packages/web-pkg/src/helpers/share/triggerShareAction.ts

```diff
@@ -1,30 +1,18 @@
-import { SpaceResource } from '@ownclouders/web-client/src'
-import {
-  aggregateResourceShares,
-  ShareResource,
-  ShareStatus
-} from '@ownclouders/web-client/src/helpers/share'
+import { ShareResource } from '@ownclouders/web-client/src/helpers/share'
 import { HttpError } from '@ownclouders/web-client/src/errors'
 import { OwnCloudSdk } from '@ownclouders/web-client/src/types'
 
+/** @deprecated */
 export async function triggerShareAction({
   resource,
   status,
-  hasResharing,
-  hasShareJail,
   client,
-  hidden = undefined,
-  spaces = [],
-  fullShareOwnerPaths = false
```

Comment:
Reminds me - have you tried this with fullShareOwnerPaths? 🙈 👀

Reply:
Not yet, it will probably break eventually with the transition to sharing NG (or tbh I think the first PR already broke it 😅 ). But I'd like to get this transition going first, especially because code paths like these will change in a few days anyway.

I added it to the epic though: https://github.com/owncloud/web/issues/10418